station/submitobs.py

```diff
@@ -56,29 +62,81 @@ def get_tle(sat_name: str, date: datetime.datetime) -> Optional[List[str]]:
     except:
         return None
 
+def get_mime_type(filename: str) -> str:
+    """
+    Returns the MIME type for known and unknown file formats.
+
+    Parameters
+    ==========
+    filename: str
+        name of the file to be sent
+    return:
+        MIME type as str
+    """
+
+    # This is a list of file types that possibly could make sense in the Svarog project.
+    # When adding new types, don't forget to update ALLOWED_FILE_TYPES in
+    # server/app/controllers/receive.py
+    known_types = {
+        "csv": "text/plain",
+        "gif": "image/gif",
+        "jpeg": "image/jpeg",
+        "jpg": "image/jpeg",
+        "json": "application/json",
+        "log": "text/plain",
+        "png": "image/png",
+        "svg": "image/svg+xml",
+        "txt": "text/plain",
+        "wav": "audio/wav"
+    }
+
+    txt = filename.lower()
+    offset = txt.rfind('.')
+    ext = txt[offset+1:] if offset != -1 else ""
+    return known_types[ext] if ext in known_types else "application/octet-stream"
+
 def submit_observation(data: SubmitRequestData):
     '''
-    Submit observation to content server.
-    '''
-    _, filename = os.path.split(data.image_path)
+    Attempts to submit the observation to the content server.
 
+    Parameters
+    ==========
+    data: SubmitRequestData
+        Observation to be sent (image, sat_name, etc.)
+    return:
+        None if successful or description of what went wrong
+    '''
     form_data = {
         "aos": data.aos.isoformat(),
         "tca": data.tca.isoformat(),
         "los": data.los.isoformat(),
         "sat": data.sat_name,
-        "notes": data.notes,
-        "rating": data.rating
+        "rating": data.rating,
+        "config": data.config
     }
 
     tle = get_tle(data.sat_name, data.aos)
     if tle is not None:
         form_data["tle"] = tle
 
-    file_obj = open(data.image_path, 'rb')
+    # Now process the files
+    files = {}
+    cnt = 0
     body: Dict[str, Any] = {
-        "file": file_obj
     }
+    for f in data.image_path:
```

Comment:
`image_path` is a singular verb. But it's a list of paths now. Also, these paths mustn't be image paths.
In this context (reading a file), the `f` is confusing. The `f` means a file descriptor. It would help if you used a `path` verb.

Reply:
renamed as suggested.